enterprise_access/apps/api/v1/views/subsidy_access_policy.py

```diff
@@ -248,6 +250,21 @@ class SubsidyAccessPolicyRedeemViewset(UserDetailsFromJwtMixin, PermissionRequir
     permission_required = 'requests.has_learner_or_admin_access'
     http_method_names = ['get', 'post']
 
+    @classmethod
+    @functools.lru_cache(maxsize=None)
+    def get_subsidy_client(cls):
+        """
+        A request-cached EnterpriseSubsidyAPIClient instance.
```

Comment:
FWIW, I think `lru_cache` will cache for the lifetime of the process, not the request lifetime.

Reply:
Interesting, that does seem to match up with my experience with these having lasting memory between unit tests, which in theory should destroy and create new instances of these views and models.  I'll update all the docstrings for now to say that they're process-cached since I think there's enough of a combination of sentiment and evidence to support the claim.  Note that `@cached_property` will have the same behavior since it's effectively backed by `lru_cache`.